README.md

```diff
@@ -0,0 +1,62 @@
+# Star rating
+This widget lets users rate an object with stars from 1 to 5.
+
+## Features
+* Rate an object with stars
+* Display average rating
+* Display rating in whole and half stars
+* Execute a microflow when the rate is changed
+
+## Dependencies
+Mendix 7.1
+
+## Demo project
+Try our the demo project https://rating100.mxapps.io with username: x/y/z  and password: 1
+Editing of a rate is possible if the logged in user owns that vote
+
+## Usage
+This widget requires a context.
+
+### Security Configuration
+
+Security is a mandatory feature for rating and is considered seriously for proper use of this widget in a mendix project.
+ - Model Configuration ![1](./assets/domain_model.jpg)
```

Comment:
Only use full url's https://github.com/FlockOfBirds/star-rating/blob/feature/mendixwrapper/assets/domain_model.jpg

Else the App store documentation URL's will break.